apis/addonconfigs/csi/v1alpha1/awsebscsiconfig_types.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// AwsEbsCSIConfigSpec defines the desired state of AwsEbsCSIConfig
+type AwsEbsCSIConfigSpec struct {
+	AwsEbsCSI AwsEbsCSI `json:"awsEBSCSIDriver"`
+}
+
+// AwsEbsCSIConfigStatus defines the observed state of AwsEbsCSIConfig
+type AwsEbsCSIConfigStatus struct {
+	// Name of the secret created by csi controller
+	//+ kubebuilder:validation:Optional
+	SecretRef *string `json:"secretRef,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// AwsEbsCSIConfig is the Schema for the awsebscsiconfigs API
+type AwsEbsCSIConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   AwsEbsCSIConfigSpec   `json:"spec,omitempty"`
+	Status AwsEbsCSIConfigStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// AwsEbsCSIConfigList contains a list of AwsEbsCSIConfig
+type AwsEbsCSIConfigList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []AwsEbsCSIConfig `json:"items"`
+}
+
+// AwsEbsCSI is the Schema for the awsebscsiconfigs API
+type AwsEbsCSI struct {
+	// The namespace csi components are to be deployed in
+	// +kubebuilder:validation:Optional
```

Comment:
I noticed in the AwsEbsCSI package readme in tce the namespace part is required
https://github.com/vmware-tanzu/community-edition/tree/main/addons/packages/aws-ebs-csi-driver/1.8.0

Reply:
thanks !